lib/features/home/home_screen/best_collections/ui/best_collections_widget.dart

```diff
@@ -0,0 +1,243 @@
+import 'package:brush_strokes/features/home/best_collections_screen/best_collections_screen.dart';
+import 'package:brush_strokes/features/home/collection_screen/collection_screen.dart';
+import 'package:brush_strokes/features/home/home_screen/best_collections/bloc/best_collections_bloc.dart';
+import 'package:brush_strokes/models/collections/collection.dart';
+import 'package:brush_strokes/repositories/featured_collections_repository.dart';
+import 'package:brush_strokes/theme/colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:shimmer/shimmer.dart';
+
+class BestCollectionWidget extends StatelessWidget {
+  const BestCollectionWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider<BestCollectionsBloc>(
+          create: (BuildContext context) =>
+              BestCollectionsBloc(FeaturedCollectionsRepository()),
+        ),
+      ],
+      child: _blocBody(),
+    );
+  }
+
+  Widget _blocBody() {
+    return BlocProvider(
```

Comment:
No need to create another BestCollectionsBloc here, since you've already done it above - it will be accessible to all child widgets.